docs/howto/build-guides/platforms.rst

```diff
@@ -0,0 +1,169 @@
+Build platform-specific charms
+==============================
+
+A ``charmcraft.yaml`` defines where charms are built and where charms are run
```

Comment:
```suggestion
A charm's ``charmcraft.yaml`` file determines which silicon architectures it builds on and which it can run on
```

Reply:
It defines more than that. New language coming up.